tests/console/repo_orphaned_packages_check.pm

```diff
@@ -0,0 +1,37 @@
+# SUSE's openQA tests
+#
+# Copyright © 2018 SUSE LLC
+#
+# Copying and distribution of this file, with or without modification,
+# are permitted in any medium without royalty provided the copyright
+# notice and this notice are preserved.  This file is offered as-is,
+# without any warranty.
+
+# Summary: Check for packages orphaned from present repos
+# Maintainer: Michal Nowak <mnowak@suse.com>
+
+use base 'consoletest';
+use strict;
+use testapi;
+use utils 'zypper_call';
+
+sub run {
+    select_console 'root-console';
+
+    # Pipe-separated list of packages we whitelist for orphan checking
+    my $orphan_whitelist = get_var('ORPHAN_WHITELIST', 'gpg-pubkey');
+    zypper_call('ref');
```

Comment:
If this is moved after the `zypper ref` module, this could be removed.

Reply:
I'd rather not count on test order.

Reply:
Agree with @mnowaksuse here, as we cannot tight the sequence we can forget about the order and shuffle tests so that it breaks again. I would keep it here.